core/src/test/java/apoc/export/csv/ImportCsvTest.java

```diff
@@ -245,6 +248,25 @@ public void testDefaultTypedField() {
         assertThat(names, Matchers.containsInAnyOrder("Jane", "John"));
     }
 
+    @Test
+    public void testCsvWithoutIdField() {
+        TestUtil.testCall(
+                db,
+                "CALL apoc.import.csv([{fileName: $file, labels: ['Person']}], [], $config)",
+                map(
+                        "file", "file:/personsWithoutIdField.csv",
+                        "config", map()
+                ),
+                (r) -> {
+                    assertEquals(2L, r.get("nodes"));
+                    assertEquals(0L, r.get("relationships"));
+                }
+        );
+
+        List<String> names = TestUtil.firstColumn(db, "MATCH (n:Person) RETURN n.name AS name ORDER BY name");
+        assertThat(names, Matchers.containsInAnyOrder("Jane", "John"));
```

Comment:
nit: you can either remove `ORDER BY` or `InAnyOrder` but it looks a little bit strange to keep both